src/PasteClient.ts

```diff
@@ -15,6 +16,14 @@ export default class PasteClient {
     this.apiKey = apiKey;
   }
 
+  /**
+   * set the API domain
+   * @param {string} domain The domain of your reverse proxy server.
+   */
+  async setDomain(domain = "pastebin.com"): Promise<void> {
+    this.domain = domain;
+  }
```

Comment:
```suggestion
  setDomain(domain = "pastebin.com"): void {
    this.domain = domain;
  }
```

This can be just a regular function :)